components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx

```diff
@@ -106,7 +121,7 @@ export default class CustomTermsOfServiceSettings extends AdminSettings {
         }
     };
 
-    handleAPIError = (err, callback, config) => {
+    handleAPIError = (err: any, callback: (() => void) | undefined, config?: Props['config']) => {
```

Comment:
To avoid the undefined, doesn't make more sense to have callback as possibly undefined? (i.e. `callback?: () => void`)

Reply:
Added, thanks for the tip